languages/clojure/exercises/concept/basic-functions/.doc/hints.md

```diff
@@ -0,0 +1,20 @@
+### 1. Create a function that returns 42
```

Comment:
I really like the hints! They are very well written.

Reply:
Thanks! 😊